SimG4Core/Application/python/g4SimHits_cfi.py

```diff
@@ -132,7 +132,7 @@
         ConfGlobalMFM = cms.PSet(
             Volume = cms.string('OCMS'),
             OCMS = cms.PSet(
-                Stepper = cms.string('G4TDormandPrince45'),
+                Stepper = cms.string('CMSTDormandPrince45'),
```

Comment:
@VinInn , there is no regression in this PR. Even if the difference is small we cannot make this change now in the master, because 12_4_X and 12_5_X are production releases for start of Run3. We should plan to this switch in the next cycle mid August.

Reply:
up to the management to decide.

Reply:
They already postponed gamma general process (~2% speedup) after positive PdmV validation.

Reply:
your proposal is just to revert this line and merge?

Reply:
Yes, it is more practical to revert this line. We may modify issue #38473 adding optimization of field to the list of pending updates for the new release.